.github/workflows/release.yml

```diff
@@ -55,6 +55,8 @@ jobs:
         with:
           node-version: '16'
           registry-url: 'https://registry.npmjs.org'
+      # TODO: Remove later, when `npm 8.5.4` is built in.
+      - run: npm i -g npm@^8.5.4
```

Comment:
This allow me to remove the --no-package-lock, which actually generate a more accurate package-lock. (still some stuff to fix, but all minor, can wait)